### ISSUE TITLE
Error atropos trim using multiple threads

### ISSUE THREAD

**Issue by ekopylova** [jdidion/atropos#30]
Hello,

I used ```atropos trim``` with ```--threads 4``` and received an error shown below. When I run the same command without the threads option, everything runs well.

```atropos trim  --info-file info_file --no-default-adapters --known-adapters-file sequencing_adapters.fa --input1 SRR5829902_1.fastq.gz --input2 SRR5829902_2.fastq.gz --trim-n -q 20 --minimum-length 90 --maximum-length 101 --max-n 1 --preserve-order --report-file atropos_report_file --log-file atropos_logfile --output SRR5829902_1_atropos_trim --paired-output SRR5829902_2_atropos_trim --threads 4```
```
Process Result process:
Traceback (most recent call last):
  File "/Users/ekopylova/miniconda/envs/atropos/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/Users/ekopylova/miniconda/envs/atropos/lib/python3.5/site-packages/atropos/commands/trim/__init__.py", line 874, in run
    num_batches if num_batches > 0 else None)
  File "/Users/ekopylova/miniconda/envs/atropos/lib/python3.5/site-packages/atropos/commands/trim/__init__.py", line 781, in finish
    self.writers.close()
AttributeError: 'ResultProcess' object has no attribute 'close'
```

Let me know if you require further info. Thanks!

Jenya

**Comment by jdidion** [comment]
Thanks for the report. It looks like this is a bug with the --preserve-order option. It should work if you run it without that.

I'm working on fixing the bug now.

**Comment by jdidion** [comment]
Fixed in version 1.1.9
